db/migrate/20201021152850_add_index_to_statistics_announcement_publication_id.rb

```diff
@@ -0,0 +1,8 @@
+class AddIndexToStatisticsAnnouncementPublicationId < ActiveRecord::Migration[5.1]
+  def change
+    # rubocop:disable Rails/BulkChangeTable
+    remove_index :statistics_announcements, name: "index_statistics_announcements_on_publication_id", column: :publication_id
+    add_index :statistics_announcements, :publication_id, unique: true
+    # rubocop:enable Rails/BulkChangeTable
```

Comment:
I think rubocop is suggesting this:

```suggestion
    change_table :statistics_announcements do |t|
      t.remove_index name: "index_statistics_announcements_on_publication_id", column: :publication_id
      t.index :publication_id, unique: true
    end
```

Reply:
Ah, that's what I was looking for! I was confused by the fact that the normal methods are `remove_index` and `add_index`, in the context of `change_table` there's `remove_index` but no `add_index`... 🤔